index.js

```diff
@@ -19,6 +19,32 @@ const getWidth = stream => {
 	return columns;
 };
 
+const getHeight = stream => {
+	const {rows} = stream;
+
+	if (!rows) {
+		return 20;
+	}
+
+	return rows;
+};
+
+const cropLinesFromFront = (text, noOfLines) => {
+	const lines = text.split('\n');
+	return lines.slice(noOfLines).join('\n');
```

Comment:
This doesn't handle ansi escapes. Imagine you have a multiline string that is colored blue, if you crop away half of the lines, it will no longer be colored, as the ansi escape that gives it its color is cropped away.

https://github.com/chalk/slice-ansi